action.yml

```diff
@@ -61,6 +61,9 @@ outputs:
 
       This URL will be valid for as long as the artifact exists and the workflow run and repository exists. Once an artifact has expired this URL will no longer work.
       Common uses cases for such a download URL can be adding download links to artifacts in descriptions or comments on pull requests or issues.
+  artifact-digest:
+    description: >
+      SHA-256 digest for the artifact that was just uploaded. Empty if the artifact upload failed.
```

Comment:
Can you also add this to the merge sub-action as well? https://github.com/actions/upload-artifact/blob/main/merge/action.yml